packages/grid/src/stories/grid.stories.tsx

```diff
@@ -0,0 +1,122 @@
+import { Grid } from '../grid';
+
+export default {
+	title: 'Grid',
+	component: Grid,
+	parameters: {
```

Comment:
```suggestion
	tags: [ 'autodocs' ],
	parameters: {
```

in case we want the Docs page
<img width="1047" alt="image" src="https://github.com/user-attachments/assets/a250c39b-680b-4e3e-a0d2-010755b37a96" />